<launch>

<arg name="map_size_x" default="50.0"/>
<arg name="map_size_y" default="50.0"/>
<arg name="map_size_z" default=" 8.0"/>

<arg name="start_x" default=" -25.0"/>
<arg name="start_y" default=" -25.0"/>
<arg name="start_z" default=" 0.0"/>

<arg name="resolution" value="0.5" />

<arg name="run_a_star" default="true"/>
<arg name="run_jps"    default="false"/>

  <node pkg="graph_searcher" type="graph_search_node" name="graph_search_node" output="screen" required = "true">
      <remap from="~waypoints"       to="/waypoint_generator/waypoints"/>
      <remap from="~map"             to="/random_forest/global_map"/> 

      <param name="occ_map/origin_x" value="$(arg start_x)"/>
      <param name="occ_map/origin_y" value="$(arg start_y)"/>
      <param name="occ_map/origin_z" value="$(arg start_z)"/>

      <param name="occ_map/map_size_x" value="$(arg map_size_x)"/>
      <param name="occ_map/map_size_y" value="$(arg map_size_y)"/>
      <param name="occ_map/map_size_z" value="$(arg map_size_z)"/>

      <param name="occ_map/resolution" value="$(arg resolution)" type="double"/>

      <param name="search/run_a_star" value="$(arg run_a_star)"/>
      <param name="search/run_jps"    value="$(arg run_jps)"/>
  </node>
</launch>